sound/soc/sof/topology.c

```diff
@@ -1258,6 +1261,13 @@ static int sof_widget_load_pga(struct snd_soc_component *scomp, int index,
 		goto err;
 	}
 
+	/* set default max volume values to control */
```

Comment:
default 0dB?

Reply:
In theory 0 dB could be out of min..max range

vol = min(vol_0db, vol_max);

Reply:
ok. but so what is max and where do I get it :)  ?

Reply:
@singalsu I know left it to 0db. I can do later the min(0 or max), but I need to change a lot to get the max from topology.